src/tutorials/CliSK/2 - Creating my first Client-Side Konnector (CliSK)/2.2 - CliSK architecture.md

```diff
@@ -0,0 +1,16 @@
+##### How it works ?
+
+The CliSK contains two webviews. 
+- One called `pilot`
+- Second called `worker`
+We will use those name during the tutorial, as it is part of mandatory understanding to make your konnector.
+
+###### Pilot
+The `pilot` webview is here to ensure the communication between the worker (navigating the web) and the Cozy App. It "pilots" (clever, right ?) the worker's actions and give back the result to the Cozy App.
```

Comment:
We should also say that the pilot is here to keep the state of the konnector execution